Fix issue when initializing DB based resource group manager (#22891)
prestodb/presto: presto-docs/src/main/sphinx/admin/resource-groups.rst

.. code-block:: text

    resource-groups.configuration-manager=db
    resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups?user=<user>&password=<password>

Replace ``<user>`` and ``<password>`` with the actual username and password

With the Database Resource Group Manager, changes to the configuration in the database take effect immediately
and do not require a restart of the Presto server. This allows for more flexibility and
dynamic changes to the resource group configurations.

> The resource group configuration must be populated through tables

Author: "from the" tables would change the meaning. We want to convey that the tables created should be used to populate the resource group configuration. If you feel we can rephrase it better, please let me know.

> The resource group configuration must be populated through tables

Reviewer: No, your explanation makes sense. No change needed here, thanks!

* ``jmxExport`` (optional):  If set to ``true``, the statistics of the resource group will be exported via JMX.
  Defaults to ``false``.

* ``perQueryLimits`` (optional): specifies max resources each query in a
  resource group may consume before being killed. These limits are not inherited from parent groups.
  May set three types of limits:

>     * ``executionTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)

Reviewer: ```suggestion
    * ``executionTimeLimit`` (optional): Specify an absolute value (for example, ``1h``)
```
This comment applies to the three sub-items in this list - please make the same changes you make to the other two items in the list (``totalMemoryLimit`` and ``cpuTimeLimit``).

* Do not use Latin abbreviations such as i.e.. For context and suggestions, see the GitLab documentation style guide recommended word list entry for [i.e.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#ie) and [e.g.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#eg). 

* Please change the indentation of the three items in the sublist to remove the vertical bar. Use the same indentation as the sublist in ``schedulingPolicy`` that begins with ``fair`` and ends with ``query_priority`` (``query_priority`` is visible in the top of the screenshot). 
<img width="792" alt="Screenshot 2024-06-03 at 9 54 27 AM" src="https://github.com/prestodb/presto/assets/7013443/ed6951fb-f52e-4030-8940-f3714d61ec42">

* ``clientTags`` (optional): List of tags. To match, every tag in this list must be in the list of
  client-provided tags associated with the query.

* ``selectorResourceEstimate`` (optional): Resource Group Selection based on resource estimates.
    - ``executionTime``
    - ``cpuTime``

>     - ``peakMemory``

Reviewer: Does ``peakMemory`` relate to ``totalMemoryLimit``? ``executionTimeLimit`` and ``executionTime``, and ``cpuTimeLimit`` and ``cpuTime`` make sense to me, but ``peakMemory`` and ``totalMemoryLimit`` not being the same I found a little confusing. 

Nit: earlier, the items are listed in the order ``executionTimeLimit``, ``totalMemoryLimit``, ``cpuTimeLimit``. 
Suggest ordering the concepts of the two lists in the same order to help the reader, for example ordering this list as ``executionTime``, ``peakMemory``, ``cpuTime``.

Author: > Does peakMemory relate to totalMemoryLimit? executionTimeLimit and executionTime, and cpuTimeLimit and cpuTime make sense to me, but peakMemory and totalMemoryLimit not being the same I found a little confusing.

So, they are not directly related to each other. The first section about Resource Group Properties (`executionTimeLimit`, `totalMemoryLimit`, `cpuTimeLimit`) is to set limits for a resource group. This means any query that exceeds these limits will be killed. 

The second section under selector rules (`executionTime`, `peakMemory`, `cpuTime`) is used for determining which resource group a query should be directed to. These are resource estimates that can be passed by the user as a request header(`X-Presto-Resource-Estimate`) or a command line argument(`--resource-estimates`) if the user has an idea of how much resources a query might use.

* ``clientInfo`` (optional): String to match against client info.

* ``principal`` (optional): This is a regular expression that matches the principal who is submitting the query.

> * ``schema`` (optional): This matches the session schema of the query

Reviewer: ```suggestion
* ``schema`` (optional): This matches the session schema of the query.
```
Nit, punctuation.

Database Resource Group Manager
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

.. code-block:: sql

    -- global properties

>     INSERT INTO resource_groups_global_properties (name, value) VALUES ('cpu_quota_period', '1h');

Reviewer: No change suggested, I just wanted to say this is a good code example.
